roles/os_hardening/defaults/main.yml

```diff
@@ -385,3 +385,57 @@ os_selinux_enabled: true
 # The values must be inside the 1000-999999999 range.
 os_sha_crypt_min_rounds: "640000"
 os_sha_crypt_max_rounds: "640000"
+
+os_mnt_boot_dir_mode: '0700'
+os_mnt_boot_enabled: false
+os_mnt_boot_src: ''
+os_mnt_boot_options: 'rw,nosuid,nodev,noexec'
+os_mnt_boot_filesystem: 'ext4'
+
+os_mnt_dev_dir_mode: '0755'
+os_mnt_dev_enabled: true
```

Comment:
Since almost all mounts are disabled by default, we should disable these, too.

Reply:
Only my background: `/dev` is a mountpoint on most/every linux distributions and its a good idea, to support users to get a out of the box hardening with good settings. Thats the way, I would expect how this collections works (per default, good settings without making my system unusable).

But sure, you are the collection owner. If you want to disable it, because all others are set to `false`, I can check again an set everywhere `false`

Reply:
I like the idea of hardening `/dev` and `/dev/shm` by default. I believe these mount points are present on every Linux distribution that we support and the chosen options for `/dev/shm` seem reasonable and are supported by hardening guideline from CIS (I could find no guideline for `/dev` so there might be some further research in order). So I vote for `true`.

While looking at this, I believe there is still something wrong with the `filesystem` and 'src' variables. Also `ext4` seems wrong in any case here.

Reply:
Thanks @rndmh3ro @schurzi for your support!

Did not find any suggestions in CIS either. We do this on our environment and we do not have any hassle with it.

The filesystem depends on every linux distribution. If you want, I can check `/boot` on several distributions. I am not sure, but on RHEL7, I thought it was ext4 - RHEL8 it is xfs.
I haven't tested it, yet. Is Ansible able to autodetect the filesystem? If it is a feature of Ansible, I can configure an empty string except it is manually set of the user.